### DISCUSSION TITLE
Zed Editor Won't Open

### DISCUSSION THREAD

**Discussion by anchapin** [zed-industries/zed#6491] [Feature Requests]
After the first time I closed Zed, I can no longer open it. When I try to open it, I see the icon in my dock with the dot under it but no window opens and when I right-click and select "Show All Windows" I see that there are no windows available.

**Comment by jansol** [comment]
The menu bar should change to that of Zed when you click on the dock icon and you should be able to use File > New Window.

**Comment by fotonick** [comment]
I had this happen to me where invoking Zed from the commandline for one specific directory wouldn't work, but all other directories would.

Root cause: I was plugged into an external monitor and had switched it to a different input, so just couldn't see my window. D'oh.

Solution: Unplug external monitor OR with Zed active (but no window visible), go to Window → Move to Built-in Retina Display.

**Comment by anchapin** [comment]
Well, I just tried to reproduce this error again to see if one of the proposed solutions would help and I'm not seeing this issue anymore.
